ceph/ceph_admin/apply.py

```diff
@@ -126,6 +126,9 @@ def apply(self: ServiceProtocol, config: Dict) -> None:
             return
 
         if not check_service_exists(
-            self.installer, service_name=service_name, service_type=self.SERVICE_NAME
+            self.installer,
```

Comment:
I think similar changes needed at the `ceph_admin/helper.py:905`.

Reply:
done.